### PR TITLE
Portfolio-Lars Söderman

### PR DESCRIPTION
**Pull request by lars-soderman**
🚀🚀🚀

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+</head>
+<body>
+    <header>
+        <div class="container">
+
+            <!-- <img src="./resources/header-bg.svg" alt="background image"> -->
+            <div class="social-media-icons">
+                <a href="https://github.com/lars-soderman"><img src="resources/github.svg" alt=""></a>
+                <a href="https://www.linkedin.com/in/lars-soderman/"><img src="resources/linkedin.svg" alt=""></a>
+                <a href="https://stackoverflow.com/users/14960150/lars-s%c3%b6derman"><img src="resources/stack-overflow.svg" alt=""></a>
```
Comment (KarolCislowski) [inline index.html]: It would be nice to add some hover effects to make your visitor subconscious want to click this link 🧙🏻

File: index.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="for-more">
+        <h2>For more</h2>
+        <div class="social-media-icons bottom">
+            <a href="https://github.com/lars-soderman"><img src="resources/github.svg" alt=""></a>
+            <a href="https://www.linkedin.com/in/lars-soderman/"><img src="resources/linkedin.svg" alt=""></a>
+            <a href="https://stackoverflow.com/users/14960150/lars-s%c3%b6derman"><img src="resources/stack-overflow.svg" alt=""></a>
```
Comment (KarolCislowski) [inline index.html]: Icons seem to be not perfectly centralized.

**Review by KarolCislowski — APPROVED** [review]
It is a very nice portfolio page 🦄  
I left below a shortlist of fixes needed to fit it better to the guidelines. Please make them soon 😸

**Comment by lars-soderman** [comment]
Hey Karol! Thanks for feedback, if you want to have another look i have adjusted the social media icons and also added some hover effects!

**Review by KarolCislowski — APPROVED** [review]
Great work!

**Pull request closed without merging**